lib/node_modules/@stdlib/blas/base/saxpy/manifest.json

```diff
@@ -233,7 +261,9 @@
         "-lblas"
       ],
       "libpath": [],
-      "dependencies": []
+      "dependencies": [
+        "@stdlib/strided/base/min-view-buffer-index"
```

Comment:
@aman-095 I believe you are missing `@stdlib/blas/base/shared`. This also needs to be fixed in `daxpy`.

Reply:
I think this skipped notice as we don't explicitly test CBLAS.

Reply:
This applies for other configurations in this file.

Reply:
Yes, I think I missed that. Updated.